turkce/catalla-kaz/metali-cek/iple-in/iple-in.md

```diff
@@ -1,6 +1,10 @@
 Iple inerken karanlığın bir taraftan aydınlandığını gördün. 
+
 Meğerse ipin aşağıdaki kısmı yanmaya başlamış. 
+
 İpin yanıp biteceğini görünce iki opsiyon aklına gelir: 
-- ne beklediğini bilmeden karanlığa atlamak veya
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.